tools/rum-models-generator/Sources/RUMModelsGeneratorCore/RUM/RUMSwiftTypeTransformer.swift

```diff
@@ -94,7 +94,7 @@ internal class RUMSwiftTypeTransformer: TypeTransformer<SwiftType> {
 
         var `struct` = `struct`
         `struct`.name = format(structName: `struct`.name)
-        `struct`.properties = try `struct`.properties.map { try transform(structProperty: $0) }
+        `struct`.properties = try `struct`.properties.map { try transform(structProperty: $0) }.filter { property in property.name != "customTimings" }
```

Comment:
Maybe add back the `TODO: RUMM-1000` here?

Reply:
oh yes definitely, thanks 🙏